### ISSUE TITLE
Nit: unreleased documentation on `main` leads to confusion

### ISSUE THREAD

**Issue by bw** [workos/authkit-nextjs#156]
**Describe the bug**

OK this is a bit of a nitpick but I went down a 20-30 minute rabbit hole because of it. It would be helpful for me (and I am sure others) if unreleased versions of the code are not on the main branch. Thanks for all the great work on the library otherwise!

**To Reproduce**

The README.md discusses an `onSuccess` option in `HandleAuthOptions`:

```
export const GET = handleAUth({
  onSuccess: async ({ oauthTokens }) => {
    await saveTokens(oauthTokens);
  },
});
```

However, this isn't released yet (the latest version as of writing is v0.16.2).

**Expected behavior**

Unreleased changes should be published to a different, non-main branch.

**Comment by chamini2** [comment]
This happened to me too, so not so nitpicky imo.

**Comment by mthadley** [comment]
Thanks all for raising! Sorry about the inconsistency. We've released a new version ([`v0.17.0`](https://www.npmjs.com/package/@workos-inc/authkit-nextjs/v/0.17.0)) which includes the `onSuccess` option.
